src/hevm/src/EVM.hs

```diff
@@ -624,18 +624,18 @@ exec1 = do
         0x09 -> stackOp3 (const g_mid) (\(x, y, z) -> mulmod x y z)
 
         -- op: LT
-        0x10 -> stackOp2 (const g_verylow) $ \(x, y) -> ite (x .< y) 1 0
+        0x10 -> stackOp2 (const g_verylow) $ \(x, y) -> iteWhiff "<" (x .< y) x y
```

Comment:
why change from `1 0` to `x y` here?

Reply:
`iteWhiff` returns 0 or 1, but with the whiff built out of x and y